tests/parsing/errors/typeDef/typeDef.res

```diff
@@ -1,3 +1,18 @@
+// The name must start with a lowercase
```

Comment:
Would you add this at the end of the file or in a new file?
It will make it easier to follow what's changed in the diff.

Reply:
Sure. Done.